README.md

```diff
@@ -30,6 +30,9 @@ These need to be added to a .env file and include:
 ```
     * Mongo_Url
     * Mongo_db
+    * Github:
+        *client_Id
+        *client_secret_m10
```

Comment:
You should provide more information - currently when registering their OAuth application in GitHub, people will have to read the codebase to work out the correct callback URL.